content/en/aws/kms/index.md

```diff
@@ -0,0 +1,123 @@
+- Symmetric encryption keys - such keys consist of just one key. Symmetric keys can **not** be used to sign and to verify messages, are only used to encrypt and to decrypt data.
+- Hash-Based Message Authentication Code (HMAC) keys, a special case of symmetric keys that are only used in creation and verification of message authentication codes (MACs) with [GenerateMac](https://docs.aws.amazon.com/kms/latest/APIReference/API_GenerateMac.html) / [VerifyMac](https://docs.aws.amazon.com/kms/latest/APIReference/API_VerifyMac.html) operations. You can read more about this type of keys [here](https://docs.aws.amazon.com/kms/latest/developerguide/hmac.html).
+
+You can check [this page](https://docs.aws.amazon.com/kms/latest/developerguide/symm-asymm-compare.html) to see which other operations are supported for each type of key. Once you know which operations you require, this reference can help to figure out which key types are suitable for your goals.
+
+# Tutorial
+
+Let's create a simple symmetric encryption key and use it to encrypt / decrypt something.
+
+A new key can be created in KMS with
+
+{{< command >}}
+$ awslocal kms create-key
+{{</ command >}}
+
+By default, this command creates a symmetric encryption key, so we do not even have to supply any additional arguments. In the output we want to pay attention to the ID of the newly created key, `010a4301-4205-4df8-ae52-4c2895d47326` in this case:
+
+```sh
+{
+    "KeyMetadata": {
+        "AWSAccountId": "000000000000",
+        "KeyId": "010a4301-4205-4df8-ae52-4c2895d47326",
+        "Arn": "arn:aws:kms:us-east-1:000000000000:key/010a4301-4205-4df8-ae52-4c2895d47326",
+        "CreationDate": 1665432947.493433,
+        "Enabled": true,
+        "Description": "",
+        "KeyUsage": "ENCRYPT_DECRYPT",
+        "KeyState": "Enabled",
+        "Origin": "AWS_KMS",
+        "KeyManager": "CUSTOMER",
+        "CustomerMasterKeySpec": "SYMMETRIC_DEFAULT",
+        "KeySpec": "SYMMETRIC_DEFAULT",
+        "EncryptionAlgorithms": [
+            "SYMMETRIC_DEFAULT"
+        ],
+        "MultiRegion": false
+    }
+}
+```
+
+If we lose the ID, we can always use
+
+{{< command >}}
+$ awslocal kms list-keys
+{{</ command >}}
+
+to list IDs and [Amazon Resource Identifiers](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) (ARNs) of all the available keys. Then, if necessary, we can also check all the details about a key with a given key ID or ARN like this:
+
+{{< command >}}
+$ awslocal kms describe-key --key-id 010a4301-4205-4df8-ae52-4c2895d47326
+{{</ command >}}
+
+We can use the key now to encrypt something. Let's say, `some important stuff`:
```

Comment:
Please remove backticks if you are not pointing to any variable, ID, code, or anything where you want the user to have a special attention. Using italics is fine.